distro/registrations.go

```diff
@@ -239,11 +240,11 @@ func Loop(config Config, errChan chan error, eventCh chan *models.Event) {
 	err := consulclient.ConsulInit(consulclient.ConsulConfig{
 		ServiceName:    applicationName,
 		ServicePort:    cfg.Port,
-		ServiceAddress: "localhost",
-		CheckAddress:   "http://localhost:48070/api/v1/ping",
+		ServiceAddress: cfg.Hostname,
+		CheckAddress:   "http://" + cfg.Hostname + ":" + strconv.Itoa(cfg.Port) + "/api/v1/ping",
 		CheckInterval:  "10s",
-		ConsulAddress:  "localhost",
-		ConsulPort:     8500,
+		ConsulAddress:  cfg.ConsulHost,
+		ConsulPort:     cfg.ConsulPort,
```

Comment:
Same here - please register to Consul during init procedure in `main` function, to keep it consistent with other init stuff (connecting to DB, enabling logger, etc...)